readthedocs/settings/base.py

```diff
@@ -741,9 +742,24 @@ def DOCKER_LIMITS(self):
     TAGGIT_TAGS_FROM_STRING = 'readthedocs.projects.tag_utils.rtd_parse_tags'
 
     # Stripe
+    # Existing values we use
     STRIPE_SECRET = None
     STRIPE_PUBLISHABLE = None
 
+    # DJStripe values
+    STRIPE_LIVE_SECRET_KEY = None
+    STRIPE_TEST_SECRET_KEY = "sk_test_x" # A default so the `checks` don't fail
+    DJSTRIPE_WEBHOOK_SECRET = None
+
+    STRIPE_LIVE_MODE = False  # Change to True in production
+    DJSTRIPE_FOREIGN_KEY_TO_FIELD = "id"
+    DJSTRIPE_USE_NATIVE_JSONFIELD = True  # We recommend setting to True for new installations
+    if not DJSTRIPE_WEBHOOK_SECRET:
```

Comment:
You may want to add this check inside a function/property

```python
@property
def DJSTRIPE_WEBHOOK_VALIDATION(self): ...
```

otherwise is evaluated at import time only and won't change its value after that.

Reply:
Yea, I thought I'd remove that..